### ISSUE TITLE
Fluid: warnings when recompiling .fl files

### ISSUE THREAD

**Issue by Albrecht-S** [fltk/fltk#543]
**Describe the bug**
Fluid outputs warnings "**_Can't chdir to (null) : Success_**" when recompiling some fluid (`.fl`) files with `fluid -c`.
The same happens when "updating" fluid files with `fluid -u`.
This does not happen for all fluid files. For examples see log below.

**To Reproduce**
1. Checkout a fresh FLTK library
2. Build the library with `make` (no options necessary)
3. Change directory to `fluid`
4. Run e.g. `./fluid -c about_panel.fl`

**Expected behavior**
Silent exit, no warnings (and files updated).

**Screenshots**
```
fltk/fluid $ for x in "-c" "-u"; do for f in *.fl;do echo; echo "--- ./fluid $x $f ---"; echo; ./fluid $x $f; done; done

--- ./fluid -c about_panel.fl ---

Can't chdir to (null) : Success
Can't chdir to (null) : Success

--- ./fluid -c alignment_panel.fl ---


--- ./fluid -c function_panel.fl ---


--- ./fluid -c print_panel.fl ---

Can't chdir to (null) : Success
Can't chdir to (null) : Success

--- ./fluid -c template_panel.fl ---


--- ./fluid -c widget_panel.fl ---


--- ./fluid -u about_panel.fl ---

Can't chdir to (null) : Success

--- ./fluid -u alignment_panel.fl ---


--- ./fluid -u function_panel.fl ---


--- ./fluid -u print_panel.fl ---

Can't chdir to (null) : Success
Can't chdir to (null) : Success

--- ./fluid -u template_panel.fl ---


--- ./fluid -u widget_panel.fl ---
```

**FLTK Version**
Please complete the following information and delete non-applicable lines:
 - Version: 1.4.0
 - Git branch: master
 - Commit: 85af0036f9fc733b2e8ad3ae62fd00bc783db738

**FLTK Configure / Build Options**
See above, just `make` with default config.

**Operating System / Platform:**
 - OS: Linux
 - OS Version: Mint 20

**Additional context**
The same happens in directory `src` for `Fl_File_Chooser.fl` but not for `Fl_Help_Dialog.fl`.

**Comment by Albrecht-S** [comment]
FWIW: this bug appeared in commit 56cba05672faa953f3e078de6a2db2992dd888b9.

**Comment by MatthiasWM** [comment]
Fixed in #545
